--- origsrc/gnome-panel-3.14.0/applets/clock/set-timezone.c	2014-10-27 10:47:36.000000000 -0500
+++ src/gnome-panel-3.14.0/applets/clock/set-timezone.c	2015-01-06 01:37:53.532067200 -0600
@@ -22,7 +22,9 @@
 #endif
 
 #include <gio/gio.h>
+#ifndef __CYGWIN__
 #include <polkit/polkit.h>
+#endif
 
 #include "set-timezone.h"
 
@@ -58,6 +60,7 @@ get_system_bus (GError **error)
   return system;
 }
 
+#ifndef __CYGWIN__
 static int
 can_set (Cache *cache, const gchar *method_name)
 {
@@ -102,11 +105,16 @@ can_set (Cache *cache, const gchar *meth
 
   return cache->value;
 }
+#endif
 
 gint
 can_set_system_timezone (void)
 {
+#ifndef __CYGWIN__
   return can_set (&can_set_timezone_cache, "CanSetTimezone");
+#else
+  return FALSE;
+#endif
 }
 
 gboolean
--- origsrc/gnome-panel-3.14.0/configure.ac	2014-10-27 10:50:24.000000000 -0500
+++ src/gnome-panel-3.14.0/configure.ac	2015-01-06 01:39:07.462955300 -0600
@@ -131,7 +131,7 @@ PKG_CHECK_MODULES(TZ, gio-2.0 >= $GLIB_R
 AC_SUBST(TZ_CFLAGS)
 AC_SUBST(TZ_LIBS)
 
-PKG_CHECK_MODULES(CLOCK, pango >= $PANGO_REQUIRED gtk+-3.0 >= $GTK_REQUIRED glib-2.0 >= $GLIB_REQUIRED gio-2.0 >= $GLIB_REQUIRED librsvg-2.0 >= $LIBRSVG_REQUIRED gweather-3.0 >= $GWEATHER_REQUIRED gnome-desktop-3.0 >= $LIBGNOME_DESKTOP_REQUIRED polkit-gobject-1 dconf >= $DCONF_REQUIRED)
+PKG_CHECK_MODULES(CLOCK, pango >= $PANGO_REQUIRED gtk+-3.0 >= $GTK_REQUIRED glib-2.0 >= $GLIB_REQUIRED gio-2.0 >= $GLIB_REQUIRED librsvg-2.0 >= $LIBRSVG_REQUIRED gweather-3.0 >= $GWEATHER_REQUIRED gnome-desktop-3.0 >= $LIBGNOME_DESKTOP_REQUIRED dconf >= $DCONF_REQUIRED)
 AC_SUBST(CLOCK_CFLAGS)
 AC_SUBST(CLOCK_LIBS)
 
