--- origsrc/gnome-panel-3.18.1/applets/clock/Makefile.am	2015-09-30 12:33:33.000000000 -0500
+++ src/gnome-panel-3.18.1/applets/clock/Makefile.am	2015-11-15 20:28:00.393012300 -0600
@@ -75,7 +75,7 @@ libclock_applet_la_LIBADD = \
 	$(NULL)
 
 libclock_applet_la_LDFLAGS = \
-	-module -avoid-version \
+	-module -avoid-version -no-undefined \
 	$(WARN_LDFLAGS) \
 	$(AM_LDFLAGS) \
 	$(NULL)
@@ -96,7 +96,7 @@ test_system_timezone_LDFLAGS = \
 	$(AM_LDFLAGS) \
 	$(NULL)
 
-APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/libclock-applet.so
+APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/cygclock-applet.dll
 
 clock_appletlibdir = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)
 clock_appletlib_LTLIBRARIES = libclock-applet.la
--- origsrc/gnome-panel-3.18.1/applets/fish/Makefile.am	2015-09-27 09:12:39.000000000 -0500
+++ src/gnome-panel-3.18.1/applets/fish/Makefile.am	2015-11-15 20:28:11.483027600 -0600
@@ -22,12 +22,12 @@ libfish_applet_la_LIBADD = \
 	$(LIBPANEL_APPLET_LIBS)
 
 libfish_applet_la_LDFLAGS = \
-	-module -avoid-version \
+	-module -avoid-version -no-undefined \
 	$(WARN_LDFLAGS) \
 	$(AM_LDFLAGS) \
 	$(NULL)
 
-APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/libfish-applet.so
+APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/cygfish-applet.dll
 
 fish_applet_libdir = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)
 fish_applet_lib_LTLIBRARIES = libfish-applet.la
--- origsrc/gnome-panel-3.18.1/applets/notification_area/Makefile.am	2015-09-27 09:13:03.000000000 -0500
+++ src/gnome-panel-3.18.1/applets/notification_area/Makefile.am	2015-11-15 20:28:25.685048400 -0600
@@ -38,7 +38,7 @@ libnotification_area_applet_la_LIBADD =
 	$(LIBPANEL_APPLET_LIBS)
 
 libnotification_area_applet_la_LDFLAGS = \
-	-module -avoid-version \
+	-module -avoid-version -no-undefined \
 	$(WARN_LDFLAGS) \
 	$(AM_LDFLAGS) \
 	$(NULL)
@@ -64,7 +64,7 @@ testtray_LDFLAGS = \
 	$(AM_LDFLAGS) \
 	$(NULL)
 
-APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/libnotification-area-applet.so
+APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/cygnotification-area-applet.dll
 
 notification_area_appletlibdir = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)
 notification_area_appletlib_LTLIBRARIES = libnotification-area-applet.la
--- origsrc/gnome-panel-3.18.1/applets/wncklet/Makefile.am	2015-09-27 09:13:17.000000000 -0500
+++ src/gnome-panel-3.18.1/applets/wncklet/Makefile.am	2015-11-15 20:28:35.185061500 -0600
@@ -34,12 +34,12 @@ libwnck_applet_la_LIBADD = \
 	$(NULL)
 
 libwnck_applet_la_LDFLAGS = \
-	-module -avoid-version \
+	-module -avoid-version -no-undefined \
 	$(WARN_LDFLAGS) \
 	$(AM_LDFLAGS) \
 	$(NULL)
 
-APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/libwnck-applet.so
+APPLET_LOCATION   = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)/cygwnck-applet.dll
 
 wnck_appletlibdir = $(pkglibdir)/$(LIBPANEL_APPLET_API_VERSION)
 wnck_appletlib_LTLIBRARIES = libwnck-applet.la
--- origsrc/gnome-panel-3.18.1/libpanel-applet/panel-applet.c	2015-10-12 16:30:39.000000000 -0500
+++ src/gnome-panel-3.18.1/libpanel-applet/panel-applet.c	2015-11-15 20:22:31.486832400 -0600
@@ -2457,7 +2457,7 @@ panel_applet_get_object_path (PanelApple
 	return applet->priv->object_path;
 }
 
-G_MODULE_EXPORT GtkWidget *
+GtkWidget *
 panel_applet_get_applet_widget (const gchar *factory_id,
                                 guint        uid)
 {
