--- origsrc/gnome-panel-3.22.0/configure.ac	2017-03-07 02:15:34.080739400 -0600
+++ src/gnome-panel-3.22.0/configure.ac	2017-03-07 02:17:57.246407700 -0600
@@ -110,7 +110,6 @@ PKG_CHECK_MODULES([PANEL], [
   pango >= $PANGO_REQUIRED
   gtk+-3.0 >= $GTK_REQUIRED
   cairo-xlib
-  gdm
   glib-2.0 >= $GLIB_REQUIRED
   gmodule-2.0 >= $GLIB_REQUIRED
   gio-2.0 >= $GLIB_REQUIRED
--- origsrc/gnome-panel-3.22.0/gnome-panel/panel-menu-items.c	2016-10-08 07:36:54.000000000 -0500
+++ src/gnome-panel-3.22.0/gnome-panel/panel-menu-items.c	2017-03-07 02:20:53.600368800 -0600
@@ -46,7 +46,9 @@
 #include <libpanel-util/panel-end-session-dialog.h>
 #include <libpanel-util/panel-session-manager.h>
 #include <libpanel-util/panel-show.h>
+#if 0
 #include <gdm/gdm-user-switching.h>
+#endif
 
 #include "menu.h"
 #include "panel-action-button.h"
@@ -383,6 +385,7 @@ panel_menu_item_activate_switch_user (Gt
 
                 if (result)
                         g_variant_unref (result);
+#if 0
         } else {
 		gboolean ret;
 
@@ -392,6 +395,7 @@ panel_menu_item_activate_switch_user (Gt
 			g_warning ("Failed to switch to greeter: %s", error->message);
 			g_error_free (error);
 		}
+#endif
 	}
 }
 
