--- origsrc/gnome-panel-3.4.1/applets/clock/set-timezone.c	2012-03-15 03:56:49.000000000 -0500
+++ src/gnome-panel-3.4.1/applets/clock/set-timezone.c	2012-05-06 21:33:34.072055200 -0500
@@ -23,7 +23,9 @@
 #endif
 
 #include <gio/gio.h>
+#if 0
 #include <polkit/polkit.h>
+#endif
 
 #include "set-timezone.h"
 
@@ -64,6 +66,7 @@ can_set (Cache *cache, const gchar *meth
 {
   guint64 now = g_get_monotonic_time ();
 
+#if 0
   if (now - cache->stamp > (CACHE_VALIDITY_SEC * 1000000))
     {
       PolkitAuthority *authority;
@@ -100,6 +103,10 @@ can_set (Cache *cache, const gchar *meth
         g_object_unref (authority);
         g_object_unref (subject);
     }
+#else
+  cache->stamp = now;
+  cache->value = 0;
+#endif
 
   return cache->value;
 }
--- origsrc/gnome-panel-3.4.1/configure.ac	2012-03-26 03:47:41.000000000 -0500
+++ src/gnome-panel-3.4.1/configure.ac	2012-05-06 21:19:04.324308500 -0500
@@ -149,7 +149,7 @@ PKG_CHECK_MODULES(TZ, gio-2.0 >= $GLIB_R
 AC_SUBST(TZ_CFLAGS)
 AC_SUBST(TZ_LIBS)
 
-PKG_CHECK_MODULES(CLOCK, pango >= $PANGO_REQUIRED gtk+-3.0 >= $GTK_REQUIRED glib-2.0 >= $GLIB_REQUIRED gio-2.0 >= $GLIB_REQUIRED librsvg-2.0 gweather-3.0 >= $GWEATHER_REQUIRED gnome-desktop-3.0 >= $LIBGNOME_DESKTOP_REQUIRED polkit-gobject-1)
+PKG_CHECK_MODULES(CLOCK, pango >= $PANGO_REQUIRED gtk+-3.0 >= $GTK_REQUIRED glib-2.0 >= $GLIB_REQUIRED gio-2.0 >= $GLIB_REQUIRED librsvg-2.0 gweather-3.0 >= $GWEATHER_REQUIRED gnome-desktop-3.0 >= $LIBGNOME_DESKTOP_REQUIRED)
 AC_SUBST(CLOCK_CFLAGS)
 AC_SUBST(CLOCK_LIBS)
 
